src/System.Reflection.Metadata/src/System/Reflection/Metadata/Ecma335/CodedIndex.cs

```diff
@@ -75,6 +64,11 @@ private enum HasCustomAttribute
             __bits = 5
         }
 
+        private static Exception UnexpectedHandleKind(HandleKind kind)
+        {
+            return new ArgumentException(SR.Format(SR.UnexpectedHandleKind, kind));
+        }
```

Comment:
Should this be Throw.UnexpectedHandleKind() for consistency with the rest of the cleanup you've done?